tests/Forms/Container.values.BackedEnum.phpt

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+use Nette\Forms\Form;
+use Nette\Utils\ArrayHash;
+use Tester\Assert;
+
+
+require __DIR__ . '/../bootstrap.php';
+
+enum TestEnum: string {
+    case CASE_1 = 'case 1';
+    case CASE_2 = 'case 2';
+    case CASE_3 = 'case 3';
+    case CASE_4 = 'case 4';
+}
+
+function createForm(): Form
+{
+	Form::initialize(true);
+
+	$form = new Form;
+	$form->addSelect('select_box', 'label S', ['case 1' => 'Case 1', 'case 2' => 'Case 2', 'case 3' => 'Case 3', 'case 4' => 'Case 4']);
```

Comment:
Now I am sad that we cannot use enums as array keys.

Reply:
Array cann't use enums like key.